runtime/lua/vim/_meta/api.lua

```diff
@@ -73,6 +73,9 @@ function vim.api.nvim__id_dictionary(dct) end
 function vim.api.nvim__id_float(flt) end
 
 --- @private
+--- NB: if your UI doesn't use hlstate, this will not return hlstate first
+--- time.
```

Comment:
This _Nota Bene_ (brief) docstring suddenly appears, which is a minor (and the only) difference resulted from this refactoring. IDK what makes a difference, but since this would be fine to include, let me keep it...